panels/applications/cc-applications-panel.c

```diff
@@ -1410,13 +1411,14 @@ populate_applications (CcApplicationsPanel *self)
       GtkWidget *row;
       g_autofree gchar *id = NULL;
 
-      if (!g_app_info_should_show (info))
+      id = get_app_id (info);
+      if (!g_app_info_should_show (info) ||
+          g_str_has_prefix (id, EOS_LINK_PREFIX))
```

Comment:
Nitpick: either keep it in a single line (preferred), or add curly braces to the if condition.